DeAceroDLabs/marketplace-frontend
# fix: [DMP-684] Fixed categories tabs spaces.

## What is done

[[DMP-684] Categories tabs spaces](https://deacero.atlassian.net/browse/DMP-684)

## How is done

- Fix to scss of `tab` , `Carousel` , `tabs` components so that space in tabs looks good.

## How to test

1. Run the project via `npm start`
2. You should see the spaces on 1299 window width okey.

Before:
![image](https://user-images.githubusercontent.com/81311646/199769291-c141248a-1da2-45a3-be40-24dd9974d5a6.png)


after:
![image](https://user-images.githubusercontent.com/81311646/199769343-1aaaf2fe-1a11-41c9-b8d7-b95aa51c833a.png)
